### PR TITLE
add following status to User object

### PR DESCRIPTION
**Pull request by kenkoooo**
User class doesn't contain the parameter "following" and I have fixed it.

### CODE REVIEW

File: twitter4j-core/src/main/java/twitter4j/User.java [hunk 1]
```diff
@@ -46,6 +46,13 @@
     String getScreenName();
 
     /**
+     * Returns true if it's following the logged in user
```
Comment (fingertricks) [inline twitter4j-core/src/main/java/twitter4j/User.java]: Suggested improved wording:- "Returns true if the authenticated user is following this user"

File: twitter4j-core/src/main/java/twitter4j/User.java [hunk 2]
```diff
@@ -46,6 +46,13 @@
...
+     *
+     * @return the following of the user
```
Comment (fingertricks) [inline twitter4j-core/src/main/java/twitter4j/User.java]: Suggest replacing "the following of the user" with "true if the authenticated user is following this user"

File: twitter4j-core/src/main/java/twitter4j/User.java (hunk 1 above)
Comment (kenkoooo) [inline twitter4j-core/src/main/java/twitter4j/User.java]: thanks for your corrections.

**Comment by yusuke** [comment]
this is a property abandoned years ago.
http://issue.twitter4j.org/youtrack/issue/TFJ-143
https://code.google.com/p/twitter-api/issues/detail?id=419
https://code.google.com/p/twitter-api/issues/detail?id=532

You cannot trust the property and Twitter4J ignores it on purpose.

**Pull request closed without merging**